help/testhelper.go

```diff
@@ -176,3 +174,16 @@ func SkipTestIfNoEnvVars(t *testing.T) TestCfg {
 
 	return TestCfg{token, owner, repo, SHA}
 }
+
+// MergeMap returns a new map, the merge of of b over a, meaning that duplicated keys in
```

Comment:
The `of of` looks weird but I cannot convince myself this is actually wrong...

Reply:
:-D

Reply:
it _is_ wrong, thanks